cmd/juju/caas/add.go

```diff
@@ -487,6 +488,9 @@ func (c *AddCAASCommand) Run(ctx *cmd.Context) error {
 	if _, ok := newCloud.Config[provider.OperatorStorageKey]; !ok {
 		newCloud.Config[provider.OperatorStorageKey] = operatorStorageName
 	}
+	if _, ok := newCloud.Config[bootstrap.ControllerServiceTypeKey]; !ok {
```

Comment:
We need to think about how to handle existing cloud definitions in clouds.yaml. Perhaps default to "LoadBalancer". The out of the box microk8s support will redo any micok8s config and correctly specify cluster ip.

Reply:
yeah, the result uses LB now by default unless it can find related definition for this cloud.